api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -42,7 +42,16 @@
 	
 	private static final Logger log = LoggerFactory.getLogger(OpenmrsConstants.class);
 	
-	public static String KEY_OPENMRS_APPLICATION_DATA_DIRECTORY = "OPENMRS_APPLICATION_DATA_DIRECTORY";
+	
+	private static String KEY_OPENMRS_APPLICATION_DATA_DIRECTORY = "OPENMRS_APPLICATION_DATA_DIRECTORY";
+	
+	public static void SET_KEY_OPENMRS_APPLICATION_DATA_DIRECTORY(String name){
+		KEY_OPENMRS_APPLICATION_DATA_DIRECTORY = name;
```

Comment:
Is this supposed to be a method? Did you take a look at our conventions at https://wiki.openmrs.org/display/docs/Java+Conventions
Was this change even part of the ticket?